python_tests/test_python_frontend.py

```diff
@@ -2408,6 +2408,25 @@ def fusion_func(fd: FusionDefinition, inps, matmul_fn) -> None:
                 fc = FusionCache.get()
                 fc.reset()
 
+    def test_unified_matmul(self):
```

Comment:
what does unified_matmul mean?

I think this is test our aten execution? can we make it explicit in the test name as well.